packages/material-ui/src/Popper/Popper.js

```diff
@@ -34,54 +34,46 @@ function getAnchorEl(anchorEl) {
 /**
  * Poppers rely on the 3rd party library [Popper.js](https://github.com/FezVrasta/popper.js) for positioning.
  */
-class Popper extends React.Component {
-  tooltipRef = React.createRef();
+const Popper = React.forwardRef(function Popper(props, ref) {
+  const {
+    anchorEl,
+    children,
+    container,
+    disablePortal,
+    keepMounted,
+    modifiers,
+    open,
+    placement: placementProps,
+    popperOptions = {},
+    transition,
+    ...other
+  } = props;
+  const tooltipRef = React.useRef();
+  const popper = React.useRef();
```

Comment:
```suggestion
  const popperRef = React.useRef();
```
I'm sorry I got brainwashed.